db/migrate/20210205132951_remove_allocation_feature_flags_from_schools.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveAllocationFeatureFlagsFromSchools < ActiveRecord::Migration[6.1]
```

Comment:
Please could you delete the columns in separate PR that is deployed later? Our migrations are ran _before_ the application is updated so some users may see errors. The database wouldn't have the field but the code would be expecting it.

Reply:
Migration removed